src/integer.c

```diff
@@ -1177,40 +1206,10 @@ Int EqInt(Obj opL, Obj opR)
 **  'LtInt' returns 1 if the integer <opL> is strictly less than the
 **  integer <opR> and 0 otherwise.
 */
+
 Int LtInt(Obj opL, Obj opR)
 {
-    Int res;
-
-    CHECK_INT(opL);
-    CHECK_INT(opR);
-
-    /* compare two small integers */
-    if (ARE_INTOBJS(opL, opR))
-        return (Int)opL < (Int)opR;
-
-    /* a small int is always less than a positive large int */
-    if (IS_INTOBJ(opL) != IS_INTOBJ(opR))
-        return (IS_INTOBJ(opL) && IS_INTPOS(opR)) ||
-               (IS_INTNEG(opL) && IS_INTOBJ(opR));
-
-    /* compare two large integers */
-    if (TNUM_OBJ(opL) != TNUM_OBJ(opR)) /* different signs? */
-        return IS_INTNEG(opL);
-
-    /* signs are equal; compare sizes and absolute values */
-    if (SIZE_INT(opL) < SIZE_INT(opR))
-        res = 1;
-    else if (SIZE_INT(opL) > SIZE_INT(opR))
-        res = 0;
-    else
-        res = mpn_cmp((mp_srcptr)CONST_ADDR_INT(opL),
-                      (mp_srcptr)CONST_ADDR_INT(opR), SIZE_INT(opL)) < 0;
-
-    /* if both arguments are negative, flip the result */
-    if (IS_INTNEG(opL))
-        res = !res;
-
-    return res;
+    return CmpInt(opL, opR) == -1 ? 1 : 0;
```

Comment:
That does not match the documentation of `CmpInt`! Correct (and also otherwise better, IMHO) would be this:
```suggestion
    return CmpInt(opL, opR) < 0;
```

Reply:
I've basically done this. I left in the redundant  ? 1 : 0 because I think it makes clear that we are into 1 for true rather than non-zero for true territory, which can be useful.